network/p2p/libp2pNode_test.go

```diff
@@ -141,18 +142,22 @@ func TestConnGater(t *testing.T) {
 	sporkID := unittest.IdentifierFixture()
 
 	node1Peers := make(map[peer.ID]struct{})
-	node1, identity1 := nodeFixture(t, ctx, sporkID, "test_conn_gater", withPeerFilter(func(pid peer.ID) bool {
-		_, ok := node1Peers[pid]
-		return ok
+	node1, identity1 := nodeFixture(t, ctx, sporkID, "test_conn_gater", withPeerFilter(func(pid peer.ID) error {
+		if _, ok := node1Peers[pid]; !ok {
+			return fmt.Errorf("id not found: %s", pid.Pretty())
```

Comment:
```suggestion
			return fmt.Errorf("peer id not found: %s", pid.Pretty())
```
We need to be more explicit about the id; the `peer_id` refers to the libp2p networking layer identifier, while the `node_id` referring to the Flow protocol identifier.